DXRVanilla/DeusEx/Classes/Mission04.uc

```diff
@@ -5,6 +5,10 @@ function Timer()
     local DXRMapVariants mapvariants;
     local string map;
 
+    if (dxr.flags.IsBingoCampaignMode() && Player.IsInState('Dying') && !dxr.flagbase.GetBool(class'DXRBingoCampaign'.static.GetBingoMissionFlag(4))) {
```

Comment:
this might be a bit confusing for players, also it won't work outside of vanilla

Reply:
How would you make it less confusing? Maybe add bingo hint text to the death screen?

Reply:
Thoughts?
![image](https://github.com/user-attachments/assets/f6519cd8-11f5-4b77-8979-261f6e51681a)

Reply:
needs more words

Reply:
"You must complete a bingo line to progress" or something lol

Reply:
How about this?
![image](https://github.com/user-attachments/assets/0b6d489f-1269-4937-a596-3c77b5b15efa)